### PR TITLE
Add:add Throwable and Exception POJOs

### PR DESCRIPTION
**Pull request by fangyincheng**

### CODE REVIEW

**Comment by wongoo** [comment]
@fangyincheng pls also add license header

File: exception.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	RegisterPOJO(&Throwable{})
+	RegisterPOJO(&Exception{})
+	RegisterPOJO(&StackTraceElement{})
+}
+
+////////////////////////////
+// Throwable interface
+////////////////////////////
+
+type ThrowableIntf interface {
```
[the remark this answers is not in the document]
Comment (fangyincheng) [reply exception.go]: It is good, I will change.

**Comment by fangyincheng** [comment]
> @fangyincheng pls also add license header

ok

**Comment by fangyincheng** [comment]
> @fangyincheng what a great pr! adding an unit test for it would be more nice.

Done. Pls review.

**Pull request merged**